Add a Team page with galleries for each theme (#421)
scipy/scipy.org: README.md

# scipy.org

> The SciPy organization website is built on the theme 

Reviewer: ```suggestion
The SciPy website is built on the 
```

```
make serve
```

## Team lists

> To update the teams, use the `team_query.py` from the theme. 

Reviewer: ```suggestion
To update the teams, run `team_query.py` provided by the theme:
```

```

> python team_query.py --org=scipy --team=scipy-core-team --title=Maintainers > maintainers.md

Reviewer: Where should maintainers.md go?  Might be easier to add a Makefile target for this.

Need to explain that they need a GH_TOKEN variable exported and [where to get that token from](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).  It needs `read:org` permissions.
